addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -212,6 +214,17 @@ export class MassMailingHtmlField extends HtmlField {
             this.onIframeUpdated();
         });
 
+        $snippetsSideBar.on('click', '.o_mobile_preview_btn', () => {
```

Comment:
Can't we use a t-on-click ?

Reply:
Seems tricky or not feasible at the moment, this button is part of `snippets.editor` which is not in owl. I've done the same thing as the `o_codeview_btn` so I suppose it's at least ok for now.